### PR TITLE
Work around javac varargs bug.

### LINKED ISSUE #6078: Vargs in when building the CFG.
Commit 45e6a2806b1323a7b49f310ad9d520fbc111309c causes the following code to crash"
```
import java.lang.invoke.MethodHandle;

public class Issue6078 {
  static void call(MethodHandle methodHandle) throws Throwable {
    methodHandle.invoke();
  }
}
```

This is a small test case for one of the crashes reported by @jacek-lewandowski in https://github.com/typetools/checker-framework/issues/6076.

### PR DESCRIPTION
**Pull request by smillst**
```
static void call(MethodHandle methodHandle) throws Throwable {
    methodHandle.invoke();
  }
``` 
in  framework/tests/all-systems/Issue6078.java

When getting the type of the method `invoke` at the callsite above, the javac compiler drops the formal parameter from the method signature.  This was causing crashing in the Checker Framework; this pull request works around this.

Fixes #6078.

### CODE REVIEW

File: dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java [hunk 1]
```diff
@@ -1294,18 +1295,6 @@ protected List<Node> convertCallArguments(
...
 
-        // NOTE: When the last parameter is a type variable vararg and the compiler
-        // cannot find a specific type use to substitute for it, the compiler will
-        // create an unbounded component type instead. For example,
-        // for the following method declaration:
-        // <T> void foo(T... ts) {}
-        // consider this method invocation:
-        // foo();
-        //
-        // At the call site, the compiler doesn't have enough information about the
```
Comment (smillst) [inline dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java:1305]: I didin't see this when debugging the example, so I deleted the comment.  (In the example, T is substituted with `Object`.)

File: dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java [hunk 2]
```diff
@@ -1248,7 +1248,8 @@ protected Node methodInvocationConvert(Node node, TypeMirror formalType) {
    * method applies to both method invocations and constructor calls.
    *
    * @param method an ExecutableElement representing a method to be called
-   * @param methodType an ExecutableType representing the type of the method call
+   * @param methodType an ExecutableType representing the type of the method call; viewpoint-adapt
```
Comment (mernst) [inline dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java:1251]: Is this a description of what is done (it is already viewpoint-adapted) or of what should be done (this method will viewpoint-adapt it)?

**Review by mernst — APPROVED** [review]
Thanks!

**Review by mernst — APPROVED** [review]
Thanks!

**Pull request merged**